### ISSUE TITLE
[Bug]: missing setting for UUID identifier (when using Pimcore\Model\Tool\UUID)

### ISSUE THREAD

**Issue by fabfischer** [pimcore/pimcore#11594]
### Expected behavior

I can set an UUID identifier in the system settings.

### Actual behavior

the documentation states that "Pimcore provides a toolkit for UUID-support.": https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/UUID_Support.html

Unfortunately the referred setting does not exist in the settings anymore.

I can set a uuid instance identifier manually though:
```pimcore:
    general:
        [...]
        instance_identifier: 'my_unique_uuid_identifier'
```

I am not sure if you just forgot the setting or `Pimcore\Model\Tool\UUID` (`models/Tool/UUID.php`) is actually not supported anymore?

### Steps to reproduce

1. check system settings. it's missing a setting for the UUID identifier

**Comment by brusch** [comment]
Thanks for reporting the issue, acutally the docs are outdated here, you need to set the identifier manually in your config.

**Comment by dvesh3** [comment]
Fixed by #11653
